build/__tools__/run-tests.sh

```diff
@@ -186,11 +186,12 @@ try pdo-create --config ${CONFIG_FILE} --ledger ${PDO_LEDGER_URL} \
     --identity user1 --save-file ${SAVE_FILE} \
     --contract mock-contract --source _mock-contract.scm
 
-# this will invoke the increment operation 50 times; the objective
+# this will invoke the increment operation n times; the objective
 # of this test is to ensure that the client touches multiple, independent
 # enclave services and pushes missing state correctly
-say increment the value with a simple expression
-for v in $(seq 1 50) ; do
+declare -i n=50
+say increment the value with a simple expression ${n} times
```

Comment:
can i suggest that if we are going to change this... its really better to do it right. Given that the point of doing multiple iterations is just to ensure that multiple eservices are contacted, why not just do a round robin on the eservice urls.

Reply:
original motivation was just to have a single place where 50 and related numbers are defined but as you point out below i failed as I must have somehow screwed up applying my old patch.  Anyway, changed it now to round-robin in updated patch.